packages/react-app/src/contexts/SettingsContext.jsx

```diff
@@ -15,6 +15,7 @@ export const SettingsProvider = ({ children }) => {
   const [xdaiRPC, setXDaiRPC] = useState('');
   const [neverShowClaims, setNeverShowClaims] = useState(false);
   const [infiniteUnlock, setInfiniteUnlock] = useState(false);
+  const [tokenListWithBalance, setTokenListWithBalance] = useState(false);
```

Comment:
This needs to be stored in localStorage as well.

Reply:
Donezo.